### DISCUSSION TITLE
Add button for pulling unit price according to data

### DISCUSSION THREAD

**Discussion by cheesestickstomatojuice** [ghostfolio/ghostfolio#3105] [Ideas]
I think it would be helpful to have a button in the "Add Activity Menu" that allows me to quickly access the unit price associated with a particular date. Currently, we must manually search through our records to find the correct entry, but having a button that automatically retrieves the price based on the inserted date would save time and effort.

**Comment by dtslvr** [comment]
Why would you want to load a historical value that most likely does not correspond to your actual purchase price?

**Reply by cheesestickstomatojuice** [reply]
If you are new to ghostfolio and you have activities that have to be imported it can be easier if you just enter the date and the price will correspond according to the date.

**Comment by chris-fj** [comment]
> Why would you want to load a historical value that most likely does not correspond to your actual purchase price?

I am new both here and to investments but I mainly invest on indexed funds and I came here exactly to ask for this. My purchase price always matches the price for a given date (since indexed funds are valued only once per day) so for me it's useful. Like the OP, when inputting new values I would just have to pick the date and the number of shares. Morningstar does this, for example.
